setup.sh

```diff
@@ -33,8 +32,8 @@ git init
 git remote add origin https://github.com/hashbang/shell-etc.git
 git fetch --all
 git reset --hard origin/master
```

Comment:
should set tracking branch too.
`git branch --set-upstream-to=origin/master master`